public/css/styles.css

```diff
@@ -59,4 +68,14 @@
     padding-left: 0px!important;
     padding-right: 0px!important;
   }
-}
+  .topClear{
```

Comment:
two classes with same name?

Reply:
i made a media breakpoint giving the two classes the same name with different property value.